Snowflake.Data/Snowflake.Data.csproj

```diff
@@ -19,7 +19,8 @@
   <ItemGroup>
     <PackageReference Include="AngleSharp" Version="0.12.1" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
-    <PackageReference Include="log4net" Version="2.0.8" />
+    <PackageReference Include="log4net" Version="2.0.11" />
+    <PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />

```

Comment:
Just curious, why this is package reference is needed?

Reply:
Hi, System.Text.RegularExpressions is a dependency of log4net
And it has a vulnerability which version 4.3.1 fixes: https://www.whitesourcesoftware.com/vulnerability-database/CVE-2019-0820